src/routes/docs/quick-starts/ruby/+page.markdoc

```diff
@@ -0,0 +1,212 @@
+---
+layout: article
+title: Start with Ruby
+description: Dive into our step-by-step guide on integrating Appwrite with your Ruby server backend application. Get your backend up and running quickly with this tutorial.
+difficulty: beginner
+readtime: 5
+back: /docs/quick-starts
+---
+Learn how to setup your first Ruby project powered by Appwrite.
+{% section #step-1 step=1 title="Create project" %}
+Head to the [Appwrite Console](https://cloud.appwrite.io/console).
+
+If this is your first time using Appwrite, create an account and create your first project.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/create-project.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/create-project.png)
+{% /only_light %}
+
+Then, under **Integrate with your server**, add an **API Key** with the following scopes.
+
+{% only_dark %}
+![Create project screen](/images/docs/quick-starts/dark/integrate-server.png)
+{% /only_dark %}
+{% only_light %}
+![Create project screen](/images/docs/quick-starts/integrate-server.png)
+{% /only_light %}
+
+| Category  {% width=120 %} | Required scopes       | Purpose |
+|-----------|-----------------------|---------|
+| Database  | `databases.write`     | Allows API key to create, update, and delete [databases](/docs/products/databases/databases). |
+|           | `collections.write`   | Allows API key to create, update, and delete [collections](/docs/products/databases/collections). |
+|           | `attributes.write`    | Allows API key to create, update, and delete [attributes](/docs/products/databases/collections#attributes). |
+|           | `documents.read`      | Allows API key to create, update, and delete [documents](/docs/products/databases/documents). |
+|           | `documents.write`     | Allows API key to read [documents](/docs/products/databases/documents). |
+
+Other scopes are optional.
+
+{% /section %}
+{% section #step-2 step=2 title="Create Ruby project" %}
+Create a Ruby CLI application.
+
+```sh
+mkdir my-app
+cd my-app
+bundle init
+```
+
+{% /section %}
+{% section #step-3 step=3 title="Install Appwrite" %}
+
+Install the Ruby Appwrite SDK.
+
+```sh
+gem install appwrite
```

Comment:
This will install the gem system wide, to add it to the local project with bundler you'll need:

```suggestion
bundle add appwrite
```

Reply:
Done